### DISCUSSION TITLE
Does this binding require that the native library be installed on system?

### DISCUSSION THREAD

**Discussion by hamza-algohary** [bailuk/java-gtk#29] [General]
Hello, I want to know whether Gtk is required to be installed in order for this library to work. This is necessary for systems where Gtk might not be installed like Windows.

**Comment by bailuk** [comment]
Hi, java-gtk requres that GTK libraries are installed.
Some documentation regarding using java-gtk on Windows: [doc/windows.md](https://github.com/bailuk/java-gtk/blob/stage/doc/windows.md)
